templates/oe_organisation/oe-organisation.html.twig

```diff
@@ -0,0 +1,37 @@
+{#
```

Comment:
TODO: move it to oe_theme_content_entity_organisation module when it will be done in OPENEUROPA-2626.